test/extended/util/framework.go

```diff
@@ -61,12 +61,35 @@ func WaitForOpenShiftNamespaceImageStreams(oc *CLI) error {
 				e2e.Logf("ImageStream Error: %#v \n", err)
 				return false
 			}
-			for tag := range is.Spec.Tags {
+			for i, tag := range is.Spec.Tags {
 				e2e.Logf("Checking tag %v \n", tag)
-				if _, ok := is.Status.Tags[tag]; !ok {
+				if _, ok := is.Status.Tags[i]; !ok {
```

Comment:
i'm unclear on why this check wasn't sufficient as is.  We should not have a status.Tags['foo'] until that tag is imported (the import might have failed, but the import needs to at least have happened).

What were you observing?

Reply:
I thought the diagnostics of imports failing could be useful down the line

I'm fine with simplifying it and revert to the old form

Reply:
I see.  hmm.  i can see that.

Reply:
reverted it in the interim before seeing your comment ... let's leave it simpler for now